triggers/service/src/main/resources/com/daml/lf/engine/trigger/db/migration/oracle/V1__Init.sql

```diff
@@ -0,0 +1,22 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- Running trigger table.
```

Comment:
We could in theory use only one migration here since nobody has any historical data on oracle. However, keeping the two in sync seems easier to understand.